ux.symfony.com/assets/styles/app.scss

```diff
@@ -11,12 +11,54 @@
 @import "../../vendor/twbs/bootstrap/scss/variables";
 @import "../../vendor/twbs/bootstrap/scss/variables-dark";
 
+// Remove Bootstrap variables/overrides
+$theme-colors: map-remove($theme-colors, "info", "light");
+
 // Boostrap remainder of required parts
 @import "../../vendor/twbs/bootstrap/scss/maps";
 @import "../../vendor/twbs/bootstrap/scss/mixins";
 @import "../../vendor/twbs/bootstrap/scss/utilities";
 @import "../../vendor/twbs/bootstrap/scss/root";
 
+// Remove multiple utilities with a comma-separated list
+// Margin
+$utilities: map-remove(
```

Comment:
I'm not sure about this kind of micro-optimizations. Reasons:

* The full Bootstrap CSS contents take only 30 KBs when compressed with br or zstd, so it's not a problem for most folks
* If we remove common utilities like `border`, they won't work when we try to use them in the future and debugging why can be frustrating.

Reply:
The bootstrap CSS files is currently 90% useless _because_ of all the utilities we do not use/need  ... and the only failure in the PageSpeed results.

<img width="1039" alt="pagespeed" src="https://github.com/user-attachments/assets/fd036ac1-30c7-4b98-b149-b253b32499d4">

As i explained some time ago, i'll rework the entire HTML of the website, the final goal is to be almost "classless", and thus facilitate content creation/aggregation (mainly for demos but also cookbook, articles, docs, etc..).

I put this work in "pause" after the "dark/light theme" debate .. but i need to move on now, and trimming down the dependencies to the only things we really use is a method i'm use to work with. It's then much easier to move/recode things :) 

So i understand your vision and would agree if this rework project was not in progress :)

Reply:
OK then. Thanks for the additional insights! And thanks as always for your work here.